sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/_transport.py

```diff
@@ -736,4 +735,13 @@ def _write(self, s):
         See http://tools.ietf.org/html/rfc5234
         http://tools.ietf.org/html/rfc6455#section-5.2
         """
-        self.ws.send_binary(s)
+        from websocket import WebSocketConnectionClosedException, WebSocketTimeoutException
+        try:
+            self.ws.send_binary(s)
+        except WebSocketTimeoutException as e:
+            raise ConnectionError('send timed out (%s)' % e)
+        except SSLError as e:
+            raise ConnectionError('send disconnected by SSL (%s)' % e)
+        except WebSocketConnectionClosedException as e:
+            raise ConnectionError('send disconnected (%s)' % e)
+            
```

Comment:
very strange - is there a newline here? it's showing up strangely.

Reply:
yes there is, pylint wanted a new line oddly enough